pkg/api/http_server.go

```diff
@@ -34,6 +34,7 @@ import (
 	"github.com/grafana/grafana/pkg/services/datasourceproxy"
 	"github.com/grafana/grafana/pkg/services/datasources"
 	"github.com/grafana/grafana/pkg/services/hooks"
+	"github.com/grafana/grafana/pkg/services/libraryelements"
 	"github.com/grafana/grafana/pkg/services/librarypanels"
```

Comment:
Is there any reason for the two services? I see in the code we treat the library panels as elements now, so a bit confused with this.

Reply:
LibraryPanelService is adds panel specific layer on top of element service which could be moved into LibraryElementService but I think this brings good seperation.

Reply:
Think I see the point, just thinking about two APIs to do pretty much the same thing. I'm considering the scenario when I want to get all library elements - which service should I call? Library elements? Library panels + Library XXX ?

This is not a strong opinion but I would make abstract Library Elements API public and create a service layer in the client that would hide the differences. But, since I'm not doing API development, this may be totally wrong and I may be missing something that I'm not familiar with :)